mlir/utils/performance/tuningRunner.py

```diff
@@ -46,10 +49,8 @@ def verifyModeFlags(verifyMode: str) -> str:
 
 #Run a gemm or conv config and verify it
 def verifyKernelWithPerfConfig(perfConfig, config, paths: Paths, options: Options) -> float:
-    print(f"Verifying with perfConfig = {perfConfig}")
+    print(f"Verifying with perfConfig = {perfConfig}", file=sys.stderr)
     config.setPerfConfig(perfConfig.strip())
-    # remove the result file generated by rocprof in previous benchmarking
```

Comment:
The next few changes have the verify execution write its results to a temporary directory, which will automatically go away at the end, to ensure that multiple threads don't step on each others' output.  We don't need to delete the last run's output here, because it already happened.